### PR TITLE
 Add infrasim compute version and the latest commit info

### PR DESCRIPTION
**Pull request by Summer-Qing**

### CODE REVIEW

**Comment by fub2** [comment]
@chenge3

File: lib/TestExecutor.py [hunk 1]
```diff
@@ -947,6 +1002,14 @@ def generate_report(self):
                 continue
             str_case_result_table += '\n' + str_line
 
+	# Prepare  Checking infrasim version and infrasim-compute latest commit info
+        str_infrasim_version = ''
+        str_infrasim_latest_commit = ''
+        for key in self.infrasim_version_dic:
+            str_infrasim_version = str_infrasim_version + 'Node({}) infrasim version: \n {} \n '.format(key, self.infrasim_version_dic[key])
+        for key in self.infrasim_compute_latest_commit_dic:
+            str_infrasim_version = str_infrasim_version + 'Node({}) Latest Git Commit: \n {} \n '.format(key,  self.infrasim_compute_latest_commit_dic[key])
```
Comment (JuneZhou19) [inline lib/TestExecutor.py]: str_infrasim_latest_commit isn't assigned.

File: lib/TestExecutor.py [hunk 2]
```diff
@@ -397,6 +402,58 @@ def launch_case(self, obj_xmlnode_next_case):
...
+            infrasim_version_ls = []
+            infrasim_compute_latest_commit = ""
+            infrasim_compute_latest_commit_ls =[]
+            for obj_node in list_node:
+                node_name = obj_node.get_name()
+                node_ip =  obj_node.get_ip()
+                node_username = obj_node.get_username()
+                node_password = obj_node.get_password()
+               
+		 # OpeN ssh client
```
Comment (JuneZhou19) [inline lib/TestExecutor.py]: Why the "N" in "OpeN" capitalized? Pls follow comment convention. So is line 445. And there is a typo in 424, 
chech ->check?

File: lib/TestExecutor.py [hunk 3]
```diff
@@ -947,6 +1008,14 @@ def generate_report(self):
                 continue
             str_case_result_table += '\n' + str_line
 
+	# Prepare  Checking infrasim version and infrasim-compute latest commit info
+        str_infrasim_version = ''
+        str_infrasim_latest_commit = ''
+        for key in self.infrasim_version_dic:
+            str_infrasim_version = str_infrasim_version + 'Node({}) infrasim version: \n {} \n '.format(key, self.infrasim_version_dic[key])
+        for key in self.infrasim_compute_latest_commit_dic:
+            str_infrasim_latest_commit = str_infrasim_version + 'Node({}) Latest Git Commit: \n {} \n '.format(key,  self.infrasim_compute_latest_commit_dic[key])
```
Comment (chenge3) [inline lib/TestExecutor.py]: str_infrasim_version should be removed here, only show the latest commit.

**Comment by JuneZhou19** [comment]
@Summer-Qing Now the version only shows in test_report.html, it would be better if it show in mail directly.

**Comment by fub2** [comment]
@Summer-Qing ??

**Comment by chenge3** [comment]
@Summer-Qing If there is no --stack during puffer run, it will fail.

**Pull request closed without merging**